tests/components/buttons.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Buttons from "../../src/demos/components/buttons";
+
+describe("components/buttons", () => {
+  it("renders correctly", () => {
+    const component = renderer.create(<Buttons />);
+    console.log(component);
```

Comment:
hehe that's nice but that's not an assertion :)

You have to make sure that this component "renders [itself] correctly". You can try to use Jest snapshot: https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest.